client/client.go

```diff
@@ -252,47 +252,48 @@ func (r *NotaryRepository) initializeRoles(rootKeys []data.PublicKey, localRoles
 
 	// we want to create all the local keys first so we don't have to
 	// make unnecessary network calls
-	for _, role := range localRoles {
+	for _, roleName := range localRoles {
```

Comment:
Try to avoid variable name changes like this. The change in name doesn't really add much to the readability of the code and creates noise in the PR.